nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -404,6 +405,13 @@ int ddxProcessArgument(int argc, char *argv[], int i)
     return 1;
   }
 
+  if (!strcmp(argv[i], "-nxrealwindow")) {
+    if (++i < argc) {
```

Comment:
Also, I suggest a better name, maybe `-addnxremotewindowprop`. At least i'd like to see "prop" in the switch name.